File: Pidev_EGOV/Pi_Egov_Java/src/esprit/egov/entities/Candidature.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package esprit.egov.entities;

/**
 *
 * @author mouad
 */
public class Candidature {
        private int id_candidature;
        private Integer entreprise;
        private String utilisateur;
        private String etat;
        private String CV;

    public Candidature() {
    }

    public Candidature(int id_candidature, Integer entreprise, String utilisateur, String etat, String CV) {
        this.id_candidature = id_candidature;
        this.entreprise = entreprise;
        this.utilisateur = utilisateur;
        this.etat = etat;
        this.CV = CV;
    }

   

    public void setCV(String CV) {
        this.CV = CV;
    }

    public String getCV() {
        return CV;
    }

    @Override
    public int hashCode() {
        int hash = 7;
        hash = 29 * hash + this.id_candidature;
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Candidature other = (Candidature) obj;
        if (this.id_candidature != other.id_candidature) {
            return false;
        }
        return true;
    }

    public int getId_candidature() {
        return id_candidature;
    }

    public void setId_candidature(int id_candidature) {
        this.id_candidature = id_candidature;
    }

    public Integer getEntreprise() {
        return entreprise;
    }

    public String getUtilisateur() {
        return utilisateur;
    }

    public void setEntreprise(Integer entreprise) {
        this.entreprise = entreprise;
    }

    public void setUtilisateur(String utilisateur) {
        this.utilisateur = utilisateur;
    } 

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

   

    @Override
    public String toString() {
        return "Candidature{" + "id_candidature=" + id_candidature + ", entreprise=" + entreprise + ", utilisateur=" + utilisateur + ", etat=" + etat + '}';
    }
        
}

File: Pidev_EGOV/EGOV_symfony/src/MyApp/UserBundle/Entity/Candidature.php
<?php

namespace MyApp\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Candidature
 *
 * @ORM\Table(name="candidature")
 * @ORM\Entity
 */
class Candidature
{
    /**
     * @var string
     *
     * @ORM\Column(name="etat", type="string", length=200, nullable=false)
     */
    private $etat;

    /**
     * @var string
     *
     * @ORM\Column(name="candidat", type="string", length=8, nullable=false)
     */
    private $candidat;

    /**
     * @var integer
     *
     * @ORM\Column(name="entreprise", type="integer", nullable=false)
     */
    private $entreprise;

    /**
     * @var string
     *
     * @ORM\Column(name="cv", type="string", length=250, nullable=false)
     */
    private $cv;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_candidature", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idCandidature;



    /**
     * Set etat
     *
     * @param string $etat
     * @return Candidature
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;

        return $this;
    }

    /**
     * Get etat
     *
     * @return string 
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * Set candidat
     *
     * @param string $candidat
     * @return Candidature
     */
    public function setCandidat($candidat)
    {
        $this->candidat = $candidat;

        return $this;
    }

    /**
     * Get candidat
     *
     * @return string 
     */
    public function getCandidat()
    {
        return $this->candidat;
    }

    /**
     * Set entreprise
     *
     * @param integer $entreprise
     * @return Candidature
     */
    public function setEntreprise($entreprise)
    {
        $this->entreprise = $entreprise;

        return $this;
    }

    /**
     * Get entreprise
     *
     * @return integer 
     */
    public function getEntreprise()
    {
        return $this->entreprise;
    }

    /**
     * Set cv
     *
     * @param string $cv
     * @return Candidature
     */
    public function setCv($cv)
    {
        $this->cv = $cv;

        return $this;
    }

    /**
     * Get cv
     *
     * @return string 
     */
    public function getCv()
    {
        return $this->cv;
    }

    /**
     * Get idCandidature
     *
     * @return integer 
     */
    public function getIdCandidature()
    {
        return $this->idCandidature;
    }
}
